src/css/parse.ts

```diff
@@ -34,7 +34,7 @@ const styleMap: Record<string, string> = {
 export function extractFontFaceData (css: string, family?: string): NormalizedFontFaceData[] {
   const fontFaces: NormalizedFontFaceData[] = []
 
-  for (const node of findAll(parse(css), node => node.type === 'Atrule' && node.name === 'font-face')) {
```

Comment:
I feel there must be another way to do this. I’ll take a look …

Reply:
okay, I think the problem here is that we mustn't assume that `font-family` comes first - we probably need somehow to validate the node before iterating over its children...

Reply:
pushed an implementation - what do you think?

Reply:
@danielroe This is awesome. I think this is good to go.